stripe/src/main/java/com/stripe/android/PaymentConfiguration.kt

```diff
@@ -38,7 +41,13 @@ data class PaymentConfiguration internal constructor(
             return prefs.getString(KEY_PUBLISHABLE_KEY, null)?.let { publishableKey ->
                 PaymentConfiguration(
                     publishableKey = publishableKey,
-                    stripeAccountId = prefs.getString(KEY_ACCOUNT_ID, null)
+                    stripeAccountId = prefs.getString(KEY_ACCOUNT_ID, null),
+                    betas = prefs.getStringSet(KEY_BETAS, emptySet())
+                        ?.fold(emptySet<StripeApiBeta>()) { acc, key ->
+                            StripeApiBeta.fromCode(key)?.let {
+                                acc.plus(it)
+                            } ?: acc
+                        }.orEmpty()
```

Comment:
maybe this is more readable:

```
                    betas = prefs.getStringSet(KEY_BETAS, emptySet())
                        .orEmpty()
                        .mapNotNull { StripeApiBeta.fromCode(it) }
                        .toSet()
```

Reply:
I don't know why I thought mapNotNull didn't exist